src/errors/error-messages.ts

```diff
@@ -45,6 +45,10 @@ export const counterPartyPoolErrorMessage = {
   softCapNotReached: 'Soft cap not reached',
   amountExceedsHardCap: 'The amount exceeds the harp cap',
   mustDepositAtLeastCommittedAmount: 'Must deposit at least committed amount',
+  poolHasNoRewards: 'No rewards have been deposited into the pool',
+  poolOwnerCannotClaimToHisOwnPool: 'Pool owner cannot claim to his own pool',
```

Comment:
his is masculine and excludes feminine. Let's rephrase it for something like:

"The pool owner cannot make a claim on their own pool."